bus_enhanced/static/src/js/bus.js

```diff
@@ -0,0 +1,63 @@
+(function() {
```

Comment:
use `openerp.bus_enhanced = function(instance) {}` and replace `openerp` by `instance` below